runtime-display/vulkan-validationlayers/spec

```diff
@@ -1,4 +1,4 @@
-VER=1.3.236
-SRCS="tbl::https://github.com/KhronosGroup/Vulkan-ValidationLayers/archive/sdk-$VER.tar.gz"
-CHKSUMS="sha256::08e308c13a762134d0a66a66de8cd5a32e4e73cfe6006247292570cc5b2b4b0b"
+VER=1.3.269
+SRCS="tbl::https://github.com/KhronosGroup/Vulkan-ValidationLayers/archive/v$VER.tar.gz"
```

Comment:
Please try not to use automatically packaged tarballs, use Git tags instead.

Reply:
Fixed.